### ISSUE TITLE
Tesseract tessdata finetuned to detect Foxhole's font

### ISSUE THREAD

**Issue by Kubuxu** [GICodeWarrior/fir#8]
I've finetuned the Tesseract model to detect better the font used by Foxhole with fewer errors (the font used by foxhole is Jost).
It for example doesn't mistake the 0 and O anymore.

If you are interested, give it a try.
The integer model is much smaller and, from my tests, as good as the float (final) model.
[engJost-integer.traineddata.gz](https://github.com/GICodeWarrior/fir/files/11029771/engJost-integer.traineddata.gz)
[engJost-final.traineddata.gz](https://github.com/GICodeWarrior/fir/files/11029772/engJost-final.traineddata.gz)

**Comment by GICodeWarrior** [comment]
Thanks! I will test this out.

What character set does this cover?

Can you share the scripts / process you used to generate these?

**Comment by GICodeWarrior** [comment]
Thanks!

Do you have some example screenshots where fir has returned incorrect results?  I'd like to add them to my test cases.

For the training data, it would be cool to generate some based on text from the Foxhole translations files.  It would be great to recognize the stockpile types in each different language.

Also, Foxhole uses Renner primarily.  I believe Jost was a previous font and is similar.
![image](https://user-images.githubusercontent.com/19803/226694297-3d492d32-1597-46b4-8986-93d36c8f1c7f.png)

**Comment by Kubuxu** [comment]
Also Renner was renamed to Jost https://www.dafont.com/renner.font

**Comment by GICodeWarrior** [comment]
Thanks, this is all helpful.  Stockpile name recognition accuracy has been a recurring issue.

This is something I'd like to work on, but it will be a few weeks before I have time to dig in.

**Comment by Kubuxu** [comment]
I have run a better finetune for 27th stockpile naming (+ asci translations of storage depot and seaport). The result is here: 
[engJost-final3.traineddata.gz](https://github.com/GICodeWarrior/fir/files/11082352/engJost-final3.traineddata.gz)
IDK how well it will work with the Sundial's naming scheme. Ours is `27[A-Z]{3-4}-[IOBAEF]\d\d`

**Comment by GICodeWarrior** [comment]
This has been integrated and solves all currently known stockpile name recognition errors.

Thanks again!
